src/hooks/use-graph-line-styles.ts

```diff
@@ -0,0 +1,33 @@
+import { GraphLineScheme } from "../settings";
+
+export interface GraphLineStyles {
+  x: undefined | string;
+  y: undefined | string;
+  z: undefined | string;
+}
+
+export const useGraphLineStyles = (
+  graphLineScheme: GraphLineScheme
+): GraphLineStyles => {
+  switch (graphLineScheme) {
+    case "accessible": {
+      return {
+        x: undefined,
+        y: "10,5",
+        z: "2,2",
```

Comment:
I would like these just to be arrays, i.e., `[10, 5]`, but when doing this, the live graph gets into a re-render loop that I don't understand. With strings and the help function below, all works as expected.

Reply:
I don't think this happens if you use useMemo in useGraphStyles. Otherwise the graph will reinit for each config change which calls setChart and so re-renders.

I think maybe setChart could just be a ref too. Will push a branch in a mo

Reply:
Fixed by using `useMemo` in `useGraphStyles` [25d028b](https://github.com/microbit-foundation/ml-trainer/pull/548/commits/25d028b019d4fc7e880a24f88b1990bc73f760a6)